cli/go.sum

```diff
@@ -0,0 +1,5 @@
+github.com/kopachevsky/cloud-foundation-toolkit v0.0.0-20190515212103-a60e275a7fa7 h1:EmKqOddOW2st/vtj3RgDoaJbll7YwkS46MIVghaejKk=
```

Comment:
just noticed, selflink should not be here

Reply:
Yeah, let's remove this or figure out why it got added.

Reply:
removed